src/main/kotlin/car/racing/Game.kt

```diff
@@ -0,0 +1,20 @@
+package car.racing
+
+class Game(drivers: Int, private val moves: Int) {
+    private val cars = Cars(drivers)
+    private val gameRecords = GameRecords()
+
+    init {
+        require(drivers > 0)
+        require(moves > 0)
+    }
+
+    fun play(): GameRecords {
+        gameRecords.reset()
+        repeat(moves) {
+            cars.runAll()
+            gameRecords.record(cars)
+        }
+        return gameRecords
+    }
```

Comment:
구현하신 모습을 봤을 때 Game이 gameRecords객체를 들고 있진 않아도 될 것 같은데 어떻게 생각하시나요?
```suggestion
    fun play(): GameRecords {
        val gameRecords = GameRecords()
        repeat(moves) {
            cars.runAll()
            gameRecords.record(cars)
        }
        return gameRecords
    }
```

Reply:
넵 맴버로 있을 필요가 없네요. GameRecords를 멤버에서 제거하도록 할께요!